sdks/python/test-suites/tox/py38/build.gradle

```diff
@@ -139,7 +139,9 @@ task archiveFilesToLint(type: Zip) {
   from ("$rootProject.projectDir") {
     include "**/*.md"
     include "**/build.gradle"
+    include '**/build.gradle.kts'
     exclude "**/node_modules/*"
+    exclude "**/.gogradle/*"
```

Comment:
if not run on a clean repo (not built before, .gogradle contains 3rd party go dependency (which is similar to node_modules that stores npm deps)